src/main/java/com/coherent/aqa/java/training/web/korobeynik/page/PostLoginPage.java

```diff
@@ -24,13 +23,16 @@ public PostLoginPage(WebDriver driver) {
         super(driver);
     }
 
-    public void makeScreenShot(){
-        screenShot.takeSnapShot(driver, SCREENSHOT_PATH);
-    }
-
     public boolean isGreetingDisplayed() {
-        wait.waitForElementDisplayed(greetingElement, Duration.ofSeconds(3));
-        return greetingElement.isDisplayed();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(3));
+        boolean isDisplayed;
+        try {
+            isDisplayed = greetingElement.isDisplayed();
+        } catch (NoSuchElementException ex) {
+            ex.printStackTrace();
+            isDisplayed = false;
+        }
```

Comment:
let's move it to some utils class, I'm sure we need this logic for some other elements